### ISSUE TITLE
[2.7.0] GraphQL: empty lines inside comments unexpectedly removed

### ISSUE THREAD

**Issue by jlowcs** [prettier/prettier#13011]
**Prettier 2.7.0**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEMCeAHOAETgDM1MBeTAcwEcAbAAwB0pNt85MBVNOAJ0zBW7dsxDjJbUQDEAlvwyZ6cqHFYjRAZTiQoAE0wAZVeuliJlzewBuAQ3oBXOEkxaY-VdU0BfVoxAAGhAIPBg5dGRQO35+CAB3AAUYhDRkEAd4uxw04IAjfjswAGs4GC07AFs4EzVkEgc+fMKSsq08Iq9kD2dguEq8uH19IaM7KGpHO2o4GQh+SrsYcIn0u0cYCCCQAAsYSvoAdR25eDQOsDgdKDRTuRtTnHSwNFyQVT5+GETC6kX6xpwYIAKzQAA8tF56HAAIqOCDwAH0JogDqKATpPJ2Qb0bZ4TywQ5yfQwHbIAAcAAZgviIHxDoU8Ol8XBPjY4NtaPD4D9QmkUHY0ABaNRDIbbfhwLkKOA-ab-JANZFAkB8Spybr8XqqqGw7kcxWA4IwbFEklkpAAJmNhTkKgmAGEIJUFSBWQBWbaOPgAFWx-KVKJszgAkgYEOUwJ4wgBBAzuHDQpF8Hw+IA)

**Input:**

```jsx
const typeDefs = gql`
  type User {
    """
    First line

    Second Line
    """
    value: String
  }
`
```

**Output:**

```jsx
const typeDefs = gql`
  type User {
    """
    First line
    Second Line
    """
    value: String
  }
`
```

**Expected behavior:**

Unless I'm mistaken, the empty comment line should not be removed by prettier.

**Comment by tubbo** [comment]
We're also seeing this on our project. Last version we were on was `v2.6.2`, so I think this might be related to https://github.com/prettier/prettier/pull/12519

![Screen Shot 2022-06-14 at 3 18 17 PM](https://user-images.githubusercontent.com/113026/173670957-ca9d6af7-fe86-42a1-9bca-d36fc058bd41.png)

**Comment by chimurai** [comment]
Caused by https://github.com/prettier/prettier/pull/12608 

You can try the fix with the `✨Try the playground for this PR✨`-link in https://github.com/prettier/prettier/pull/13013 to verify the fix.

Sorry for the inconvenience.

**Comment by fisker** [comment]
Fixed in v2.7.1 https://github.com/prettier/prettier/releases/tag/2.7.1

Thank you @chimurai
